openedx/core/djangoapps/organization_api/tests/test_views.py

```diff
@@ -0,0 +1,85 @@
+"""
+Tests Organization API.
+"""
+import json
+import unittest
+from django.conf import settings
+from django.core.urlresolvers import reverse
+from django.test import TestCase
+from oauth2_provider.tests.factories import AccessTokenFactory, ClientFactory
+
+from student.tests.factories import UserFactory
+from util import organizations_helpers
+
+
+@unittest.skipUnless(settings.ROOT_URLCONF == 'lms.urls', 'Test only valid in lms')
+class OrganizationsAPITests(TestCase):
+    """
+    Tests for the organizations API endpoints.
+
+    GET /api/organizations/v1/get_organization/org_key/
+    """
+
+    def setUp(self):
+        """
+        Test organizations API.
+        """
+        super(OrganizationsAPITests, self).setUp()
+
+        self.password = 'password'
```

Comment:
I think we don't need this, for login we can simply use, `self.user.password` for loging.